ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolLogger.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.eth.manager;
+
+import org.hyperledger.besu.ethereum.p2p.rlpx.wire.Capability;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EthProtocolLogger {
```

Comment:
What's the reason for a separate class? Doesn't this lose the context of where it was called from?

Reply:
I made it a separate class so that I can set this one to DEBUG level and EthProtocolManager to TRACE level. It's still pretty obvious from the name where it comes from. There's a similar setup in DiscoveryProtocolLogger